### ISSUE TITLE
distorted faces. What am i doing wrong? it dosnt learn tthe face

### ISSUE THREAD

**Issue by protector131090** [ostris/ai-toolkit#82]
![1723823891168__000001500_0](https://github.com/user-attachments/assets/9d779893-36fd-41c4-b86a-2b8bd53a994e)

Why am i getting weird distortion in faces? im using 24 vram config. i tried on multiple people datasets (Xl works amazing with them) but Flux dosnt learn them. it distorts faces and overtrains.

**Comment by protector131090** [comment]
job: extension
config:
  name: OPA_RANK64
  process:
  - type: sd_trainer
    training_folder: P:\DREAMBOOTH\FLUX\Opa\model
    device: cuda:0
    network:
      type: lora
      linear: 64
      linear_alpha: 64
    save:
      dtype: float16
      save_every: 200
      max_step_saves_to_keep: 40
    datasets:
    - folder_path: P:\DREAMBOOTH\FLUX\Opa\img
      caption_ext: txt
      caption_dropout_rate: 0.05
      shuffle_tokens: false
      cache_latents_to_disk: true
      resolution:
      - 512
      - 768
      - 1024
    train:
      batch_size: 1
      steps: 5000
      gradient_accumulation_steps: 1
      train_unet: true
      train_text_encoder: false
      content_or_style: balanced
      gradient_checkpointing: true
      noise_scheduler: flowmatch
      optimizer: adamw8bit
      lr: 0.0004
      ema_config:
        use_ema: true
        ema_decay: 0.99
      dtype: bf16
    model:
      name_or_path: S:\ai-toolkit\black-forest-labs\FLUX.1-dev
      is_flux: true
      quantize: true
    sample:
      sampler: flowmatch
      sample_every: 250
      width: 1024
      height: 1024
      prompts:
      - 'photo of a 20-year-old woman, medium shot, holding a coffe cup with text:
        ''OPA'' , and city in the background'
      neg: ''
      seed: 42
      walk_seed: true
      guidance_scale: 3
      sample_steps: 25
meta:
  name: OPA_RANK64
  version: '1.0'

**Comment by WarAnakin** [comment]
oh wow, that's a strange one. Only thing i can think of is changing the training type to content, see if that works

**Comment by davsharian** [comment]
it's probably overcooked, try less steps, 1500-2500 for datset of 15-20 images.

**Comment by martintomov** [comment]
did you solve this?

try:
```bash
linear: 32 # or 16
linear_alpha: 32 # or 16
steps: 2000
linear_timesteps: true

**Comment by jaretburkett** [comment]
Are you using webp images? Someone reported a bug with webp images previously. They are currently not officially supported.

**Comment by protector131090** [comment]
> did you solve this?
> 
> try:
> 
> ```shell
> linear: 32 # or 16
> linear_alpha: 32 # or 16
> steps: 2000
> linear_timesteps: true
> ```

yes. Lower the rank the better. Rank 16 with same dataset is perfect. 1024 only is even better
